roles/installer/templates/cronjobs/metrics-utility-gather.yaml.j2

```
@@ -52,9 +52,11 @@ spec:
                 name: {{ metrics_utility_secret }}
 {% endif %}
             volumeMounts:
+{% if metrics_utility_ship_target != "crc" %}
```

Shouldn't this match the conditional below?

```suggestion
{% if metrics_utility_ship_target == "directory" %}
```

---

No because this will be required when shipping the metrics from the other cronjob that uses this same template to targets other than directory (such as s3 buckets).

---

And it may be needed for crc as well... waiting for confirmation still.

---

But shouldn't the volumeMount and the volume both only be added under the same conditions?

---

I suppose until we add more shipping target options these can both be either == directory or != crc so I'll just do == directory for now until we have more targets to ship to.